lib/eChronicler_web/views/journal_entry_view.ex

```diff
@@ -0,0 +1,3 @@
+defmodule EChroniclerWeb.JournalEntryView do
+  use EChroniclerWeb, :view
+end
```

Comment:
You don't actually have a view for a single journal entry, right? I would delete this too.

Reply:
Every time I delete this my entire app stops working! I will have to investigate further

Reply:
I think Grace needs it.  I struggled with it too locally, but I think the naming is the convention for linking a view (`JournalEntryView`) to the controller (`JournalEntryController`), and then the view connects to the correct template file based on the action method called in the controller.  It's a confusing convention.